esp/logging/logginglib/logthread.cpp

```diff
@@ -181,9 +182,15 @@ bool CLogThread::queueLog(IEspUpdateLogRequest* logRequest)
 
 bool CLogThread::queueLog(IEspUpdateLogRequestWrap* logRequest)
 {
-    unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
+// ESP Log Update
+// Would be good to find a way to delay calculation of delta time in case log message is filtered
+// Look for msTick() - startTime elsewhere
+//    unsigned startTime = (getEspLogLevel()>=LogNormal) ? msTick() : 0;
+    bool isLogging = !queryLogMsgManager()->rejectsCategory(LegacyMsgCatNormal);
```

Comment:
This seems reasonable but we should avoid LegacyMsgCat if possible.
Is there a pre-existing jlog category we could use here @wangkx thoughts?

Reply:
How about InfoMsgThreshold or DebugMsgThreshold?

Reply:
@wangkx it's up to you, I believe LegacyMsgCatNormal is comprised of All Audience, and All class...
If we use InfoMsgThreshold or DebugMsgThreshold, it would be subject to be rejected based on the jlog configuration for Audience and/or class type.